assets/js/state/selectors/cluster.js

```diff
@@ -62,6 +62,8 @@ export const getClusterSapSystems = createSelector(
   }
 );
 
+export const MIXED_VERSIONS = 'MIXED_VERSIONS';
```

Comment:
Sorry! The value of this should be just `'mixed_versions'`, the contract that we have with the checks team is shaped like that :sweat_smile:

Reply:
No worries, will revert :ok_hand: